src/components/Variables/VariableInternalComment.tsx

```diff
@@ -34,6 +34,7 @@ function VariableInternalComment({ variable, update, editing, sx }: VariableInte
 
   const toggleDialog = useCallback(() => {
     setIsDialogOpen((prev) => !prev);
+    setInternalComment(variableValue?.internalComment || '');
```

Comment:
This resets internal comments to the original value every time the dialog is closed